modules/cnv-updating-access-mode-for-live-migration.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-for-live-migration_{context}"]
+= Updating access mode for LiveMigration
+
+Default storage classes use a ReadWriteOnce (RWO) access mode by default.
+When you add additional storage classes to the *ConfigMap*, you specify the
+appropriate access mode.
+
+For LiveMigration to function properly, your storage class must use the
+ReadWriteMany (RWX) access mode. Use this 
+procedure to update the access mode, if needed.
+
+.Procedure
+* Update the *ConfigMap*
```

Comment:
I guess I understand why you thought this was related to storage classes, the configmap is keyed on storage class. But the config map is just for the UI to determine if the user selects a storage class which accessMode to create the PVC with.

So the configmap basically says. If you use this storage class in the UI, then the UI will know to create the matching DataVolume (and thus PVC) with these accessModes. But from a kubernetes perspective there is no relation between storage class and accessModes.